ck.sh

```diff
@@ -56,15 +56,10 @@ case "$COMMAND" in
 				# Copy custom skin to ckeditor-dev
 				cp -r skins/moono-lexicon ckeditor-dev/skins/moono-lexicon
 
-				# Copy skin files for plugins
-				pluginsWithSkins=$(find ckeditor-dev/plugins -maxdepth 2 -mindepth 2 -type d -name skins)
-				for plugin in $pluginsWithSkins; do
-					pluginDir=$(dirname "$plugin")
-					pluginName=$(basename "$pluginDir")
-					pluginSkinPath="skins/moono-lexicon/plugins/$pluginName/skins/moono-lexicon"
-					if [[ -d "$pluginSkinPath" ]]; then
-						cp -r "$pluginSkinPath" "$pluginDir/skins"
-					fi
+				# Copy plugins
+				plugins=$(find plugins -maxdepth 1 -mindepth 1 -type d)
```

Comment:
Can we just copy plugins folder here?

`cp -r "plugins" "ckeditor-dev"`

Reply:
:+1
sure let's keep it simple